subprojects/android-test/test-inhouse-runner/build.gradle.kts

```diff
@@ -7,13 +7,13 @@ plugins {
 }
 
 dependencies {
+    api(project(":subprojects:android-test:junit-utils"))
```

Comment:
Step to check the value locally
0. Revert :subprojects:android-test:junit-utils from `api` to `implementation` in Inhouse module.
1. Run ./gradlew buildHealth
2. Find build/reports/dependency-analysis/advice-pretty.json
3. Find :subprojects:android-test:test-inhouse-runner
4. Look at dependency :subprojects:android-test:junit-utils
5. Plugin proposes from `implementation` to `api`, because inhouse ABI contains :subprojects:android-test:junit-utils